ui/k-info-panel/switch_info.kytos

```diff
@@ -72,7 +148,70 @@
          this.update_switch_content()
        }
      }
-   }
+   },
  }
 </script>
 
+<style>
+.link-table {
+  color: #ccc;
+  max-height: 250px;
+  text-align: center;
+  margin: 0 auto;
+  display: block;
+  padding: 0.5em 0 1em 0.3em;
+  font-size: .8em;
+  height: 190px;
```

Comment:
Since you have a static height you need to define an overflow in case you have too many links. You can test this case adding manually 20 more links to the table just to check how it will be rendered.

Reply:
Overflow is now supported.  Commit [24adf0a6946dcee7a18ed7afd21553a83195c085](https://github.com/kytos-ng/topology/pull/73/commits/24adf0a6946dcee7a18ed7afd21553a83195c085)